backend/src/main/java/com/github/valfink/backend/fooditem/FoodItemExceptionAuthorization.java

```diff
@@ -0,0 +1,7 @@
+package com.github.valfink.backend.fooditem;
+
+public class FoodItemExceptionAuthorization extends RuntimeException {
```

Comment:
In Java, the naming convention for exceptions is CauseException. I suggest changing this for all your exceptions to improve readability.

Reply:
Do you have a source for that? Couldn't find any infos like that.